tests/Debug/ChartInfoBuilderTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neusta\ConverterBundle\Tests\Debug;
+
+use Neusta\ConverterBundle\Debug\Builder\ChartInfoBuilder;
+use Neusta\ConverterBundle\Debug\Model\ChartInfo;
+use Neusta\ConverterBundle\Debug\Model\DebugInfo;
+use Neusta\ConverterBundle\Debug\Model\ServiceArgumentInfo;
+use Neusta\ConverterBundle\Debug\Model\ServiceInfo;
+use PHPUnit\Framework\TestCase;
+
+final class ChartInfoBuilderTest extends TestCase
+{
+    public function testSimpleRelationBetweenServices(): void
+    {
+        $serviceA = new ServiceInfo('type_a', 'MyServiceA', [
+            new ServiceArgumentInfo('reference', '@service_b'),
+        ]);
+        $serviceB = new ServiceInfo('type_b', 'MyServiceB', []);
+
+        $debugInfo = new DebugInfo();
+        $debugInfo->add('service_a', $serviceA);
+        $debugInfo->add('service_b', $serviceB);
+
+        $builder = new ChartInfoBuilder();
+        $chartInfo = $builder->buildFromDebugInfo($debugInfo);
+
+        $this->assertInstanceOf(ChartInfo::class, $chartInfo);
+        $this->assertNotEmpty($chartInfo->nodes);
```

Comment:
This assertion is a little bit loose isn't it?

Reply:
To check if it is not empty or if there is exactly one element...ok, it's different, but not loosy ;-)